SampleProject/VOKCoreDataManagerTests/ManagedObjectImportExportTests.m

```diff
@@ -674,4 +722,13 @@ - (NSArray *)customMapsArrayWithKeyPaths
              [VOKManagedObjectMap mapWithForeignKeyPath:COOL_RANCH_KEYPATH_KEY coreDataKey:COOL_RANCH_DEFAULT_KEY]];
 }
 
+- (NSArray *)customMapsArrayWithoutMicroseconds
+{
+    return @[[VOKManagedObjectMap mapWithForeignKeyPath:FIRST_NAME_DEFAULT_KEY coreDataKey:FIRST_NAME_DEFAULT_KEY],
```

Comment:
I was just mimicking what was here already. Same reason my docblocks are missing the leading `*` on each line.